config/config.go

```diff
@@ -12,6 +12,10 @@ var (
 	ServeWeb, _ = strconv.ParseBool(getEnv("SERVE_WEB", "false"))
 	// Port defines the port used by the api server
 	Port = getEnv("PORT", "8080")
+	// NoRestartOnBlacklist defines if dyno restart on blacklisted. use if you have a static ip
+	NoRestartOnBlacklist = getEnv("NO_RESTART", "false")
```

Comment:
This shouldn't be needed since the heroku package doesn't talk to heroku if the heroku specific environment variables aren't set

Reply:
yeah that was a "not so clever" way to bypass the "reboot try" and mute the log